airbyte-ci/connectors/metadata_service/lib/metadata_service/gcs_upload.py

```diff
@@ -318,14 +328,27 @@ def upload_metadata_to_gcs(bucket_name: str, metadata_file_path: Path, validator
     doc_version_uploaded, doc_version_blob_id = _doc_upload(metadata, bucket, docs_path, False, False)
     doc_inapp_version_uploaded, doc_inapp_version_blob_id = _doc_upload(metadata, bucket, docs_path, False, True)
 
-    if not validator_opts.prerelease_tag:
+    latest_uploaded, latest_blob_id = False, None
+    doc_latest_uploaded, doc_latest_blob_id = doc_inapp_latest_uploaded, doc_inapp_latest_blob_id = False, None
+    
+    release_candidate_uploaded, release_candidate_blob_id = False, None
+    
+    # Latest upload
+    # We upload 
+    # - the current metadata to the "latest" path
+    # - the current doc to the "latest" path
+    # - the current inapp doc to the "latest" path
+    if should_upload_latest:
```

Comment:
I think we'll eventually want to move this logic into a helper function so it can be accessed by the final part of the RC publishing pipeline. But all good if you want to leave it here for now.

Reply:
I envision the "final part of the RC publishing pipeline" to be orchestrated via an `airbyte-ci` command which would:
* Delete the `release_candidate/metadata.yaml` file (could be done in this package)
* Set the `isReleaseCandidate` field to `false` in `metadata.yaml` (could be done in this package)
* Push the current version with the `latest` (will likely be done directly in `airbyte-ci`)

So I don't think a change here is required: the update of the `metadata.yaml` `isReleaseCandidate` field will change the behavior of the upload function

Reply:
Okay cool, that sounds good.